tests/integration/test_search_views.py

```diff
@@ -379,3 +383,25 @@ def _debug(log_output):
 
     current_app_mock.logger.debug.side_effect = _debug
     api_client.get('/literature/facets?q=test query')
+
+
+def test_regression_author_count_10_does_not_display_zero_facet(isolated_app, api_client):
+    record_json = {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'document_type': ['article'],
+        'titles': [{'title': 'Article with 10 authors'}],
+        '_collections': ['Literature'],
+        'authors': []
+    }
+    for i in range(10):
+        record_json['authors'].append({'full_name': 'Pincopallino' + str(i)})
+
+    rec = InspireRecord.create(data=record_json)
```

Comment:
I'd use the factory for creating a record :)

Reply:
I think the factory wouldn't work here if he needs the record to be in ES as well. I am not sure though

Reply:
hmm, It sends the record on ES if you pass ``index_name`` https://github.com/inspirehep/inspire-next/blob/master/tests/integration/helpers/factories/db/invenio_records.py#L120

Reply:
yes, we looked into that once and it didn't work (when I was working on the tests in test_records_views). And then we decided it was not in the scope of the story and that we will look into it

Reply:
I confirm, my first attempt used the factory and it did not work.